pom.xml

```diff
@@ -11,7 +11,7 @@
 
     <groupId>fr.igred</groupId>
     <artifactId>omero_macro-extensions</artifactId>
-    <version>1.1.0</version>
+    <version>1.2.0-SNAPSHOT</version>
```

Comment:
`<version>1.2.0</version>` 
should be enough.

Reply:
done in https://github.com/GReD-Clermont/omero_macro-extensions/pull/10/commits/c76bd5864f85c8febd75677f0d289ae3ec3c4c20